### DISCUSSION TITLE
PlexKodiConnect for syncing clients

### DISCUSSION THREAD

**Discussion by JimmyMc123** [croneter/PlexKodiConnect#1909] [Q&A] [answered]
Hello everyone. I’m really new to Kodi, been a Plex user all my life. I got interested in Kodi mainly for video addons (like Crackle and some others). I’m trying to find a way to sync my favorites and progress across my different devices running Kodi, but as I said, just for the content I watch on Addons. Is this possible using PlexKodiConnect?

Thanks

**Comment by croneter** [comment, marked as answer]
No, PlexKodiConnect only syncs your Plex libraries to your main Kodi video library

**Reply by JimmyMc123** [reply]
Thanks a lot for the answer. I know this might not be the place, but would you happen to know any other way to sync addons across devices in Kodi?

**Reply by croneter** [reply]
No, sorry

**Reply by mash2k3** [reply]
> Thanks a lot for the answer. I know this might not be the place, but would you happen to know any other way to sync addons across devices in Kodi?

Look into kodi centralized sql db and an addon called osmosis
